README.md

```diff
@@ -12,6 +12,20 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 
 Please make sure to update tests as appropriate.
 
+### Instructions
+Clone the repository to your local machine <br>
+Install the composer dependencies by running the command ```composer install``` <br>
+
+Once this step is completed you can call the required php functions by including them in php see the sample bellow.
+
+```
+<?php
+require dirname(__DIR__) .'/vendor/autoload.php';
+
+//see test case for more details regaring examples.
```

Comment:
What do you think including a small code snippet here? It could come from the examples in the tests? Some inspiration [here](https://github.com/guzzle/guzzle).

Reply:
hy @pabloolvcastro i will look in to it and make some sample code and include it in a new section in the readme